GymBro/Views/RoutineViews/RoutineSelectionView_AddRoutineSheet.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  AddRoutineView.swift
+//  GymBro
+//
+//  Created by user236772 on 4/12/23.
+//
+
+import SwiftUI
+/// A pop-up sheet in the parent view RoutineSelectionView
+///
+/// This is a page show up after clicking "+ Add Routine" button in RoutineSelectionVIew. Customer will able to type in their desire name and create a new routine in this page.
+///
+struct RoutineSelectionView_AddRoutineSheet: View {
+    
+    
+    // MARK: PROPERTY
+    
+    @Environment(\.presentationMode) var presentationMode
+    @State var textFieldText: String  = ""
+    
+    
+    // MARK: BODY
+    
+    var body: some View {
+        NavigationView {
```

Comment:
Theoretically, we don't need NavigationView and ScrollView at the same time because MainView has one Navigation, and NavView can cover the functionality of Scrollview.

Let's discuss this in the meeting!

We need some knowledge research.